azext_iot/digitaltwins/params.py

```diff
@@ -573,9 +573,31 @@ def load_digitaltwins_arguments(self, _):
         )
         context.argument(
             "adx_table_name",
-            options_list=["--adx-table-name", "--adxt"],
-            help="Name of Azure Data Explorer table to be created. If not provided, will use the format "
-                 "adt_dh_{dt_name}_{dt_location}.",
+            options_list=["--adx-property-event-table-name", "--adx-table-name", "--adxt"],
+            help="The name of the Azure Data Explorer table used for storing updates to properties of twins and relationships.",
+            arg_group="Azure Data Explorer",
+        )
```

Comment:
yes - in this scenario, I wanted to make sure that the user understands that the original table is now "slightly different" as in it is now known as the "Property Event Table"

Ideas for cutting down the number of things to write:
- remove "-name" from the table params
- relationship -> "rel" - though this may not be a straightforward shortened form
- remove "events" as every table now has events in its name and we can try having the param help imply the events
- there is also one boolean that I really don't know how to shorten more